subsys/bluetooth/host/mesh/cfg_srv.c

```diff
@@ -1122,12 +1108,58 @@ static u8_t va_add(u8_t *label_uuid, u16_t *addr)
 		return STATUS_UNSPECIFIED;
 	}
 
+	free_slot->ref = 1;
 	free_slot->addr = *addr;
 	memcpy(free_slot->uuid, label_uuid, 16);
 
 	return STATUS_SUCCESS;
 }
 
+static u8_t va_del(u8_t *label_uuid, u16_t *addr)
+{
+	int i;
+
+	if (addr) {
+		*addr = BT_MESH_ADDR_UNASSIGNED;
+	}
+
+	for (i = 0; i < ARRAY_SIZE(labels); i++) {
+		if (!memcmp(labels[i].uuid, label_uuid, 16)) {
+			if (addr) {
+				*addr = labels[i].addr;
+			}
```

Comment:
Minor nitpick: add an empty line after the }. In general, the coding style is that there should be an empty line after }, unless there's another } on the next line.